packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +683,59 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: AuthorizeIDPParams) {
+        const privateClient = !!this.clientSecret
```

Comment:
non-blocking, it would be nice if `isPrivate` is a property of the auth class instance. You can imagine setting it from the class constructor

```js
this.isPrivate = !!this.clientSecret
```